lib/encryption/encryption.dart

```diff
@@ -100,11 +100,12 @@ class Encryption {
 
   void handleDeviceOneTimeKeysCount(
       Map<String, int>? countJson, List<String>? unusedFallbackKeyTypes) {
-    runInRoot(() => olmManager.handleDeviceOneTimeKeysCount(
+    runInRoot(() async => olmManager.handleDeviceOneTimeKeysCount(
```

Comment:
I don't get why this makes a difference 🤔 the method `handleDeviceOneTimeKeysCount` is void and not `Future<void>` anyway and not even async so we are not awaiting this anyway

Reply:
Probably a lint rule enforces this?

Reply:
Yes, that is the one drawback I found from the discarded_futures. It does warn you when using a Future in a closure, but the closure is not marked as async. However I do think the value it provides elsewhere is beneficial.

Reply:
Okay sounds fair